### PR TITLE
[CLI] Add e2e for aptos move run

### PR DESCRIPTION
**Pull request by 0xjinn**
### Test Plan

2023-06-23 11:41:27,751 - INFO - These tests passed:
2023-06-23 11:41:27,751 - INFO - test_metrics_accessible
2023-06-23 11:41:27,751 - INFO - test_init
2023-06-23 11:41:27,751 - INFO - test_account_fund_with_faucet
2023-06-23 11:41:27,751 - INFO - test_account_create
2023-06-23 11:41:27,751 - INFO - test_account_lookup_address
2023-06-23 11:41:27,751 - INFO - test_aptos_header_included
2023-06-23 11:41:27,751 - INFO - test_move_compile
2023-06-23 11:41:27,751 - INFO - test_move_compile_script
2023-06-23 11:41:27,751 - INFO - test_move_publish
2023-06-23 11:41:27,751 - INFO - test_move_run
2023-06-23 11:41:27,751 - INFO - All tests passed!

### CODE REVIEW

File: crates/aptos/e2e/cases/move.py [hunk 1]
```diff
@@ -105,3 +105,29 @@ def test_move_compile_script(run_helper: RunHelper, test_name=None):
...
+            "--function-id",
+            "default::cli_e2e_tests::mint_hero",
+            "--args",
+            "string:Boss",
+            "string:Male",
+            "string:Jin",
+            "string:Undead",
+            "string:",
+        ],
+    )
```
Comment (gregnazario) [inline crates/aptos/e2e/cases/move.py:129]: We will probably need more tests around different types as well as nested types but this is a good start.

**Review by gregnazario — APPROVED** [review]

File: crates/aptos/e2e/cases/move.py (hunk 1 above)
Comment (0xjinn) [reply crates/aptos/e2e/cases/move.py:129]: Good call! Created an issue to keep track of this as follow up - https://github.com/aptos-labs/aptos-core/issues/8827

File: crates/aptos/e2e/cases/move.py [hunk 2]
```diff
@@ -105,3 +105,29 @@ def test_move_compile_script(run_helper: RunHelper, test_name=None):
...
+
+    response = json.loads(response.stdout)
```
Comment (banool) [inline crates/aptos/e2e/cases/move.py:131]: Same feedback as with the other commands, can you please assert that it actually worked by querying the on chain state?

File: crates/aptos/e2e/local_testnet.py [hunk 3]
```diff
@@ -8,7 +8,8 @@
 import time
 
 import requests
-from common import FAUCET_PORT, METRICS_PORT, NODE_PORT, Network, build_image_name
+from common import (FAUCET_PORT, METRICS_PORT, NODE_PORT, Network,
```
Comment (banool) [inline crates/aptos/e2e/local_testnet.py:11]: Please run isort first and then black.

**Review by banool — CHANGES_REQUESTED** [review]

File: crates/aptos/e2e/local_testnet.py (hunk 3 above)
Comment (0xjinn) [reply crates/aptos/e2e/local_testnet.py:11]: Hmm I guess order matters. I just added a simple script to run linter related for e2e. 

https://github.com/aptos-labs/aptos-core/pull/8838

File: crates/aptos/e2e/cases/move.py (hunk 2 above)
Comment (0xjinn) [reply crates/aptos/e2e/cases/move.py:131]: Yep. 

Is running the `view_hero` function the best way to verify`mint_hero` successfully ran?

Comment (banool) [reply crates/aptos/e2e/cases/move.py:131]: Could be a good way to do it yeah!

**Comment by 0xjinn** [comment]
Updated e2e to run "aptos move view" to make sure mint_hero succeed by ensuring data exist on chain

**Review by banool — APPROVED** [review]

**Pull request merged**